backend/api/src/config.rs

```diff
@@ -36,7 +36,7 @@ impl Default for AppConfig {
             config_file: "config".to_string(),
             bind_address: "[::]:8080".to_string(),
             database_url: "postgres://postgres:postgres@localhost:5432/scuffle-dev".to_string(),
-            turnstile_secret_key: "6LcZ8uIUAAAAAAM8ZQXZ0Z0Z0Z0Z0Z0Z0Z0Z0Z0Z".to_string(),
+            turnstile_dummy_secret: "6LcZ8uIUAAAAAAM8ZQXZ0Z0Z0Z0Z0Z0Z0Z0Z0Z0Z".to_string(),
```

Comment:
Change the VALUE to `"turnstile_dummy_secret"` not the key.

Reply:
lmao